src/app/chat/store/chat.effects.ts

```diff
@@ -22,4 +22,17 @@ export class ChatEffects {
       );
     })
   );
+
+  @Effect({dispatch: true})
+  sendMessage = this.actions.pipe(
+    ofType(MessageActions.ActionTypes.StartSendingMessage),
+    switchMap((sendAction: MessageActions.StartSendingMessage) => {
+      return this.mesService.sendMessage(sendAction.payload).pipe(
+        mergeMap(() => [new MessageActions.MessageSendSuccess(), new MessageActions.LoadMessagesBegin()]),
```

Comment:
Let's make it more straightforward. Init LoadMessagesBegin not in `mergeMap`, but after the whole `swithMap` above.